doc/pages/blurbs.json

```diff
@@ -504,11 +505,20 @@
   },
   {
     "name": "Jeremiah Tabb",
-    "image": "/assets/contributers/jeremiah_tabb.jpg",
+    "image": "/assets/contributors/jeremiah_tabb.jpg",
     "profile": "I am a full stack engineer and composer. I contribute because mental health awareness is one of the most significant issues in our modern society, causing a lot of pain to a lot of people.",
     "location": "Boulder, CO",
     "link_name": "jollyjerr",
     "link": "https://github.com/jollyjerr",
     "social": "github"
+  },
+  {
+    "name": "Jasmine Logan",
+    "image": "/assets/contributors/jasmine_logan.png",
```

Comment:
You'll need to upload your photo :)

Reply:
i added it to the assets/images/contributors folder. is there another place it should go?

Reply:
Looks good!